api/modules/dataSyncs.js

```diff
@@ -21,6 +21,33 @@ const dataSyncs = module.exports = (() => {
         })
     }
 
+    const findContributor = async (userUrl) => {
+        return db.models.Contributor.findOne({
+            where: {
+                github_handle: userUrl
+            }
+        })
+    }
+
+    const findContributionsByGithubUrlAndUsername = async (issueUrl, userUrl) => {
+        return db.models.Contribution.findOne({
+            include: [
+                {
+                    model: db.models.Issue,
+                    where: {
+                        github_url: issueUrl
+                    },
+                },
+                {
+                    model: db.models.Contributor,
+                    where: {
+                        github_handle: userUrl
+                    },
+                }
+            ]
+        });
+    }
```

Comment:
- Move to a new module
- Use a singleton object for `params`
- `findContributions` vs `findContribution`
- Consider using `get` instead of `find` if it's more consistent with the rest of the codebase
- `findContributorByGithubUrl`

Reply:
In `modules `there are only find, create, update functions when they are used to manage information inside the database.